contracts/feature-tests/abi-tester/abi_tester_expected_main.abi.json

```diff
@@ -391,6 +391,7 @@
             "outputs": []
         }
     ],
+    "events": [],
```

Comment:
Add events in the abi tester contract to check that they are rendered correctly.